site/docs/hosting/supabase.md

```diff
@@ -49,7 +48,7 @@ bot.command("ping", (ctx) => ctx.reply(`Pong! ${new Date()}`));
 
 const handleUpdate = webhookCallback(bot, "std/http");
 
-serve(async (req) => {
+Deno.serve(async (req) => {
```

Comment:
This is wrong, right? The server handler must return a `Response`, but it looks like in `catch` it doesn't.